src/settings/Settings.js

```diff
@@ -229,6 +229,11 @@ export const SETTINGS = {
         displayName: _td('Enable automatic language detection for syntax highlighting'),
         default: false,
     },
+    "stripJpegMetadata": {
```

Comment:
Would suggest probably `stripUploadMetadata` or maybe `stripImageMetadata`: if we start stripping metadata from other image file types we'll want to use the same setting (the description should continue to say JPEG whilst that's all it does).